src/js/facility-locator/components/FacilityMap.jsx

```diff
@@ -17,6 +17,7 @@ class HowToGetHere extends Component {
       va_health_facility: 'health',
       va_cemetery: 'cemetery',
       va_benefits_facility: 'benefits',
+      vet_center: 'vet-centers',
```

Comment:
Yes, good catch. I'd like to do some clean up on this once both the Access to Care and Vet Center features are deployed..

The switch to camelCase I implemented in the Access to Care branch so once that gets in I can also implement your suggestion